e2e/reporters/reporter.js

```diff
@@ -48,14 +48,12 @@ class SnapshotAwareReporter {
     const filename = path.basename(testPath);
     const name = ancestorTitles.join(' > ');
     const statBase = {name, filename, status, title, time: duration};
-
     if (status === 'passed') return statBase;
 
-    const snapshotParts = [filename, ...ancestorTitles, title, '1-snap', 'diff'];
-    const snapshotName = snapshotParts.map(sanitize).join('-') + '.png';
+    const snapshotParts = [filename, ...ancestorTitles, title, 'diff'];
+    const snapshotName = snapshotParts.map(sanitize).join('-') + '.jpg';
```

Comment:
Please ensure you use totally the same test name builders inside the reporter and in the img serializer.
Sure it could be copy-passed for now (until it is written on different language platforms)

Reply:
> Please ensure you use totally the same test name builders inside the reporter and in the img serializer. Sure it could be copy-passed for now (until it is written on different language platforms)

Speaking of different language platforms. Did some investigation, and ts script are still not supported

Reply:
I'd suggest a separate module written on ES (JS) but with a ts declaration file, so you will be free to use both sources.